src/NovelRT.DotNet/NovelRT.DotNet.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Update="NovelRT.DotNet.runtimeconfig.json" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="NovelRT.DotNet.runtimeconfig.json" CopyToOutputDirectory="PreserveNewest" CopyToPublishDirectory="PreserveNewest" />
```

Comment:
So this is probably(?) why the runtimeconfig wasn't properly being copied from the MSBuild side. I thought it'd get picked up by DefaultNoneItems or whatever the property is in the .NET SDK but apparently not :shrug: